### ISSUE TITLE
Add spinning icon next to npm under dependencies node in Visual Studio

### ISSUE THREAD

**Issue by mitulsuthar** [aspnet/Mvc#7801]
While creating a new ASP.NET CORE 2.0 MVC application with Angular, the npm package restore takes a lot of time. It takes around 3 minutes. Is this expected? Am I doing something wrong? I have latest VS 15.7.1 update installed and have windows 10 anniversary update. If this is normal then I have a feature request.

Feature request.
What if just like the latest spinning icon that were added for test explorer window, if there was a similar spinning icon next to npm under dependencies tree. It will give a visual indication that something is still processing. There is a processing icon at the bottom left but it is not obvious as a first time user. When I see yellow mark, it feels like I have done something wrong.

**Comment by pranavkm** [comment]
@barrytang, would this go through VS feedback?

**Comment by barrytang** [comment]
Yes, to report VS issues, use the [Report a Problem](https://developercommunity.visualstudio.com/) tool.
